### ISSUE TITLE
end of life with backup

### ISSUE THREAD

**Issue by bengsoon** [argilla-io/argilla-plugins#26]
A wild idea sparked when @davidberenstein1957 shared the cool end of life plugin #7 with me:
-  to create a plugin that automatically serialize a particular dataset push to dvc or git at the end of life, or when the dataset is fully labeled.

**Comment by davidberenstein1957** [comment]
Thanks @bengsoon! It looks great, however, some additional config for DVC would also need to be done:)

**Comment by davidberenstein1957** [comment]
Also, I'm not sure if DVC is the most practical option, but we will figure something out.

**Comment by bengsoon** [comment]
yeah i was thinking of that too -- I think DVC will add a few more lines of code :sweat_smile:
